dags/openshift_nightlies/tasks/utils/rosa_post_install.py

```diff
@@ -0,0 +1,56 @@
+from os import environ
+from openshift_nightlies.util import var_loader, executor, constants
+from openshift_nightlies.tasks.index.status import StatusIndexer
+from openshift_nightlies.models.release import OpenshiftRelease
+from openshift_nightlies.models.dag_config import DagConfig
+
+import json
+from datetime import timedelta
+from airflow.operators.bash import BashOperator
+from airflow.operators.subdag import SubDagOperator
+from airflow.models import Variable
+from airflow.models import DAG
+from airflow.utils.task_group import TaskGroup
+from kubernetes.client import models as k8s
+
+
+
+
+class Diagnosis():
+    def __init__(self, dag, config: DagConfig, release: OpenshiftRelease):
+
+        # General DAG Configuration
+        self.dag = dag
+        self.release = release
+        self.config = config
+        self.release_name = release.get_release_name(delimiter="-")
+
+        self.aws_creds = var_loader.get_secret("aws_creds", deserialize_json=True)
+
+        # Specific Task Configuration
+        self.vars = var_loader.build_task_vars(
+            release=self.release, task="install")
+
+        self.all_vars = {
+            **self.vars,
+            **self.aws_creds,
+        }
+
+        # Dump all vars to json file for Ansible to pick up
+        with open(f"/tmp/{self.release_name}-postinstall-task.json", 'w') as json_file:
+            json.dump(self.all_vars, json_file, sort_keys=True, indent=4)
+
+        super().__init__()
+
+        self.exec_config = executor.get_executor_config_with_cluster_access(self.config, self.release, executor_image="airflow-managed-services")
+
+    def _get_rosa_postinstallation(self, operation="postinstall", trigger_rule="all_success"):
+        return BashOperator(
```

Comment:
since this is running a python command can you use the `PythonOperator` in Airflow to just call the python function directly? You should be able to pass the vars dictionary as a function argument instead of having to write it to a file and stuff.

Reply:
I dont have experience with Python Operator and this is working fine as bashoperator. changing to pythonoperator will requires another week of development and testing, and we need this PR merged asap, so l will left as bashoperator and lets evaluate changing to pythonoperator on future release